src/App.js

```diff
@@ -1,13 +1,51 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodList } from './components/GoodList';
 // or
 // import * as goodsAPI from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const App = () => {
+  const [goods, setGoods] = useState([]);
+
+  return (
+    <div className="App">
+      <h1 classNmae="App__name">Dynamic list of Goods</h1>
+      <div className="App__goods goods">
+        <GoodList
+          goods={goods}
+          className="goods__list"
+        />
+        <div className="goods__button">
+          <button
+            type="button"
+            className="button is-primary"
+            onClick={async() => setGoods(await getAll())}
+          >
+            Load all goods
+          </button>
+
+          <button
+            type="button"
+            className="button is-primary"
+            onClick={async() => setGoods(await get5First())}
+          >
+            Load 5 first goods
+          </button>
+
+          <button
+            type="button"
+            className="button is-primary"
+            onClick={async() => setGoods(await getRedGoods())}
```

Comment:
let's create a separate function handleClick, which will get a callback and then utilize it
also save the result in a variable and only then pass it to setGoods